src/app/clients/clients-view/client-actions/accept-client-transfer/accept-client-transfer.component.ts

```diff
@@ -68,16 +68,13 @@ export class AcceptClientTransferComponent implements OnInit {
    */
   submit() {
     const acceptClientTransferFormData = this.acceptClientTransferForm.value;
-    const locale = this.settingsService.language.code;
     const dateFormat = this.settingsService.dateFormat;
     const prevTransferDate: Date = this.acceptClientTransferForm.value.transferDate;
     if (acceptClientTransferFormData.transferDate instanceof Date) {
       acceptClientTransferFormData.transferDate = this.dateUtils.formatDate(prevTransferDate, dateFormat);
     }
     const data = {
       ...acceptClientTransferFormData,
-      dateFormat,
```

Comment:
Client transfer request has date field. 
Do we really no need of `dateFormat` and `locale`?

Reply:
Yes, This issue and the above were found in other implementation using the latest fineract version